update_scripts.sh

```diff
@@ -8,25 +8,6 @@ echo
 echo "Fetch the latest scripts from GitHub - Navid200"
 echo
 
-if [ ! -s /srv/repo ]  # Create the file containing the repository name if nonexistent.
-then
-cat > /srv/repo << EOF
-nightscout-vps
-EOF
-fi
-if [ ! -s /srv/brnch ]  # Create the file containing the branch name if nonexistent.
-then
-cat > /srv/brnch << EOF
-vps-1
-EOF
-fi
-if [ ! -s /srv/username ]  # Create the file containing the user name if nonexistent.
-then
-cat > /srv/username << EOF
-jamorham
-EOF
-fi
-
```

Comment:
Why was this part deleted?

Reply:
The update_scripts utility was meant to update scripts and nothing else when I first created it.
 
When I recorded the repository details in files inside the srv directory, I also added this behavior to the update_scripts file.  That was unnecessary.
I should never have done that.  Those files are updated in bootstrap.  That's the only place they should be updated.

That's the reason for removing the creation of those files from the update_scripts.sh file.